src/Components/Cards/Cards.tsx

```diff
@@ -0,0 +1,199 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { useState, useRef, useEffect } from "react";
+import moment from "moment";
+import thumbnail from "../../assets/thumbnail.svg";
+import dot from "../../assets/dot.svg";
+import star from "../../assets/star_empty.png";
+import formatDate from "../../utils/formatDate";
+import styles from "./Cards.module.css";
+import Modal from "../Modal/Modal";
+
+function Cards({ items }) {
```

Comment:
아래처럼 props에 대한 타입들도 지정해주면 좋을 것 같아요. 

```suggestion

interface CardsProps { 
  items : Card[]
}

function Cards({ items } : CardsProps) {
```